### PR TITLE
BXMSDOC-3981-master: Update the Managing & monitoring business processes document for PAM 7.4

### PR DESCRIPTION
**Pull request by kaldesai**
[Epic](https://issues.jboss.org/browse/BXMSDOC-3980)
[JIRA](https://issues.jboss.org/browse/BXMSDOC-3981)
[Managing & monitoring business processes in BC doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-3981-PAM-MMBP/#process-instances-default-filters-proc)

Please review the new sections 3.3 and 4.2 from the document. Also, I have restructured the chapters 3 and 4.

### CODE REVIEW

File: doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc [hunk 1]
```diff
@@ -11,12 +11,13 @@ Users with the `process-admin` role can view and manage all user tasks from the
 
 You can open, view, and modify the details of a task, such as the due date, the priority, or the task description, by clicking a task in the list. The following tabs are available in the task page:
 
-image::admin-and-config/task-details.png[Task details]
+//image::admin-and-config/task-details.png[Task details]
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc]: should be removed?

File: doc-content/enterprise-only/processes/creating-new-process-instance-list-proc.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [id='creating-creating-new-process-instance-list-proc-{context}']
-= Creating a custom process instance list
+= Creating custom process instance filters
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/processes/creating-new-process-instance-list-proc.adoc]: This one would be related to advanced filters, so I would suggest something like: _Filtering process instances using advanced filters_ same applies for _4.2. Creating custom task filters_

Comment (kaldesai) [reply doc-content/enterprise-only/processes/creating-new-process-instance-list-proc.adoc]: Ack

File: doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc [hunk 3]
```diff
@@ -13,10 +13,11 @@ You can open, view, and modify the details of a task, such as the due date, the
 
 image::admin-and-config/task-details.png[Task details]
 
-* *Work*: Displays basic details about the task and the task owner. You can click the *Claim* button to claim the task. To undo the claim process, click the *Release* button.
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc]: Added that part again. Mistakenly removed.

File: doc-content/enterprise-only/admin-and-config/viewing-task-variables-basic-filters-in-bc-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,15 @@
+[id='viewing-task-variables-basic-filters-in-bc-proc']
+= Viewing task variables using basic filters
+
+{CENTRAL} provides basic filters to view task variables in *Manage Tasks* and *Task Inbox*. You can view the task variables of the task as columns using *Show/hide columns*.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Tasks* or  go to *Menu* -> *Track* -> *Task Inbox*.
+. On the *Task Inbox* page, click the filter icon on the left of the page to expand the *Filters* panel
```
Comment (csherrar) [inline doc-content/enterprise-only/admin-and-config/viewing-task-variables-basic-filters-in-bc-proc.adoc]: Should the customer be able to choose their method?
If yes, then the next line should be changed to reflect this and the subsequent steps checked to make sure they correctly reflect the page the user is on.

s/On the *Task Inbox* page,/On the *Manage Tasks* page or the *Task Inbox* page,

File: doc-content/enterprise-only/processes/viewing-process-instances-variables-advanced-filters-in-bc-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the advanced filters icon to expand the *Advanced Filters* panel.
+. In the *Advanced Filters* panel, enter the name and description of the filter, and click *Add New*.
+. From the *Select column* list, select the `processId` attribute. The value will change to *processId != value1*.
+. From the *Select column* list, select *equals to* for the logical query.
+. In the text field, enter the name of the process id.
+. Click *Save* and the filter is applied on the current process instance list.
+. Click *Show/hide columns* on the upper right of the process instances list and the process instance variables of the specified process id will be displayed.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/viewing-process-instances-variables-advanced-filters-in-bc-proc.adoc]: s/Click *Show/hide columns* on the upper right of the process instances list and the process instance variables of the specified process id will be displayed./Click *Show/hide columns* on the upper right of the process instances list. The process instance variables of the specified process id is displayed.

File: doc-content/enterprise-only/processes/viewing-process-instances-variables-basic-filters-in-bc-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+{CENTRAL} provides basic filters to view process instance variables. You can view the process instance variables of the process as columns using *Show/hide columns*.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the filter icon on the left of the page to expand the *Filters* panel.
+. In the *Filters* panel, select the *Definition Id*.
++
+The filter is applied on the current process instance list.
+. Click *Show/hide columns* on the upper right of the process instances list and the process instance variables of the specified process id will be displayed.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/viewing-process-instances-variables-basic-filters-in-bc-proc.adoc]: s/Click *Show/hide columns* on the upper right of the process instances list and the process instance variables of the specified process id will be displayed./Click *Show/hide columns* on the upper right of the process instances list. The process instance variables of the specified process id is displayed.

Comment (kaldesai) [reply doc-content/enterprise-only/processes/viewing-process-instances-variables-basic-filters-in-bc-proc.adoc]: Done.

File: doc-content/enterprise-only/processes/viewing-process-instances-variables-advanced-filters-in-bc-proc.adoc (hunk 5 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/viewing-process-instances-variables-advanced-filters-in-bc-proc.adoc]: Done.

**Pull request merged**